source/contributing/style-guide.md

```diff
@@ -16,119 +15,104 @@ The same holds for communication directed to contributors and maintainers:
-- Do: Use the imperative voice when giving instructions.
-    - Example: "Add the `python310` package to `buildInputs`"
-    - Don't: Use a narrative voice e.g. "Now we'll add the `python310` package to `buildInputs`".
+- Follow the evidence-based [plain language guidelines](https://www.plainlanguage.gov/guidelines/)
 
-[plain language guidelines]: https://www.plainlanguage.gov/guidelines/
+- Use imperative in direct instructions
 
-### Use inclusive language
+- Avoid narrative or discursive style
 
-Adapted from [Contributor Covenant] and [The Carpentries Code of Conduct]:
+## Be truthful
 
-- Use welcoming and inclusive language
-- Show empathy and respect towards other people
-- Be respectful of different viewpoints and experiences
-- Give and gracefully accept constructive criticism
-- Focus on what is best for the community
-
-Avoid idioms as they can be hard to understand for non-native English speakers.
+> Incorrect documentation is often worse than no documentation.
+>
+> — attributed to [Bertrand Meyer](https://web.archive.org/web/20080706015334/https://www.eskimo.com/~hottub/software/programming_quotes.html)
 
-Don't try to be funny.
-Humor is highly culturally sensitive.
-At best, jokes may obfuscate the relevant instructions.
-At worst, jokes may offend readers and invalidate our effort to help them learn.
+- Describe the subject factually
 
-Don't use references to popular culture.
-What you may consider well-known may be entirely obscure and distracting to people from different backgrounds.
+  Avoid value judgement or emotional appeal.
+  We don't know how and for which purpose the tools we document are going to be used, and our goal is not to advertise but to teach and inform.
 
-[Contributor Covenant]: https://github.com/EthicalSource/contributor_covenant/blob/cd7fcf684249786b7f7d47ba49c23a6bcb3233eb/content/version/2/1/code_of_conduct.md
-[The Carpentries Code of Conduct]: https://github.com/carpentries/docs.carpentries.org/blob/4691971d9f49544054410334140a4fd391a738da/topic_folders/policies/code-of-conduct.md
+- Use and provide evidence
 
-### Be concise
+  Provide links to other resources, such as the reference manuals or source code.
+  This helps guide readers on their learning journey, and discover related relevant information.
+  The Nix documentation team explicitly encourages you to update or restructure the manuals where appropriate, to improve the overall experience.
 
-> I would have written a shorter letter, but I did not have the time.
->
-> — [Blaise Pascal][Blaise Pascal]
+  Relying on reference documentation as much as possible also reduces the maintenance burden, as many key facts can be documented in a single place.
 
-Readers' time and attention is limited.
+- Show fully working examples
 
-Take the time to be extraordinarily respectful with their cognitive resources.
+  Code samples must always be working correctly when run as given.
+  Nix provides us with everything needed to make this happen.
```

Comment:
```suggestion
  Nix provides us with everything needed to make this happen.
  
  If you are going to present an example that does not work, e.g. to illustrate a common mistake, explain it beforehand.
  Otherwise readers may get stuck trying to make the code work before reading on.
```

Reply:
I think we could even make use of something like [this](https://myst-parser.readthedocs.io/en/latest/syntax/optional.html#block-attributes) to mark intentionally broken code examples.

Reply:
@asymmetric Thanks! Updated the suggestion.